docs/en/operations/settings/merge-tree-settings.md

```diff
@@ -1140,3 +1140,9 @@ When `cache_populated_by_fetch` is disabled (the default setting), new data part
 If enabled, `cache_populated_by_fetch` will instead cause all nodes to load new data parts from storage into their cache without requiring a query to trigger such an action. 
 
 Default value: 0.
+
+## add_implicit_sign_column_constraint_for_collapsing_engine
+
+If true, add an implicit constraint for the `sign` column of CollapsingMergeTree tables so only valid values (`1` and `-1`) are allowed.
```

Comment:
```suggestion
If true, adds an implicit constraint for the `sign` column of CollapsingMergeTree table to allow only valid values (`1` and `-1`).
```

Reply:
Thanks, I've changed it and also included that this covers VersionedCollapsingMergeTree as well.